netbench/cdk/src/main/java/com/aws/NetbenchAutoApp.java

```diff
@@ -62,30 +67,48 @@ public static void main(final String[] args) {
         ec2InstanceType = (ec2InstanceType == null) 
             ? "c5n.xlarge" 
             : ec2InstanceType.toLowerCase();
+
+        String scenario = (String)app.getNode().tryGetContext("scenario");
+        scenario = (scenario == null) 
+            ? ""
```

Comment:
what does it mean to have a null/empty scenario? Is that an error?

Reply:
I think this accidentally got left in from an earlier merge with a different local branch, I'll remove it from this commit. I decided to handle the scenario file outside of the cdk app for now.